### PR TITLE
Add GraalVM JDK with native image capability

### PR DESCRIPTION
**Pull request by PerfectSlayer**
This PR adds GraalVM JDK with native-image capability in order to start supporting native-image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,6 +8,10 @@ RUN set -eux; \
     sudo mkdir -p /usr/lib/jvm/oracle8; \
     curl -L --fail "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=246284_165374ff4ea84ef0bbd821706e29b123" | sudo tar -xvzf - -C /usr/lib/jvm/oracle8 --strip-components 1
 
+# GraalVM with native image support
+FROM ghcr.io/graalvm/graalvm-ce:22.2.0 AS graalvm-native-image
```
Comment (mcculls) [inline Dockerfile:12]: could we use a tag which includes the JDK version and also include that JDK version in the aliased name?

that way we could have native installs of both 11 and 17

Comment (PerfectSlayer) [reply Dockerfile:12]: Yes, I will switch it from `22.2.0` to `ol8-java17-22.2.0` (the `java17-22.2.0` from the official document does not seem to exist on the GH registry).
And I will install the JDK11 variant too.

File: Dockerfile [hunk 2]
```diff
@@ -8,6 +8,13 @@ RUN set -eux; \
     sudo mkdir -p /usr/lib/jvm/oracle8; \
     curl -L --fail "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=246284_165374ff4ea84ef0bbd821706e29b123" | sudo tar -xvzf - -C /usr/lib/jvm/oracle8 --strip-components 1
 
+# GraalVM with native image support
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.2.0 AS graalvm-native-image-jdk11
```
Comment (mcculls) [inline Dockerfile:12]: `ol8-java11-22.2.0` also exists if we want to keep the same version of "OracleLinux" as below

**Review by mcculls — APPROVED** [review]
LGTM

**Pull request merged**